test/table.test.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import {mount, configure} from 'enzyme';
+import Table from '../src/client/Table';
+import Adapter from 'enzyme-adapter-react-16';
+
+configure({ adapter: new Adapter() });
+
+describe('When scenario', () => {
+  const wrapper = mount(<Table />);
```

Comment:
I suggest to render your component by `shallow` instead of `mount` if we don't need to full render or verify the correctness of `lifecycle`.